.github/workflows/rust.yml

```diff
@@ -33,14 +33,16 @@ jobs:
   build-linux:
     runs-on: ubuntu-latest
     env:
+      BYOND_MAJOR: 513
+      BYOND_MINOR: 1521
```

Comment:
still 🤔

Reply:
vOv

Reply:
literally using that version for no reason, even the /tg/ dockerfile is newer...

Reply:
https://github.com/tgstation/rust-g/issues/56